### PR TITLE
Benchmark: Allow run without measure

### PR DESCRIPTION
**Pull request by maxep**
### What and why?

Allow running benchmark scenario without measures so we can debug and define s8s tests without collecting data.

### Review checklist
- [ ] Feature or bugfix MUST have appropriate tests (unit, integration)
- [ ] Make sure each commit and the PR mention the Issue number or JIRA reference
- [ ] Add CHANGELOG entry for user facing changes

### CODE REVIEW

File: BenchmarkTests/BenchmarkTests.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme [hunk 1]
```diff
@@ -53,8 +53,8 @@
       <EnvironmentVariables>
          <EnvironmentVariable
             key = "BENCHMARK_RUN"
-            value = "instrumented"
-            isEnabled = "YES">
+            value = "metrics"
+            isEnabled = "NO">
```
Comment (mariedm) [inline BenchmarkTests/BenchmarkTests.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme:57]: Will there still be instrumented scenario?

**Review by mariedm — DISMISSED** [review]

File: BenchmarkTests/BenchmarkTests.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme (hunk 1 above)
Comment (maxep) [reply BenchmarkTests/BenchmarkTests.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme:57]: This change was not intented, I reverted 👍

Comment (mariedm) [reply BenchmarkTests/BenchmarkTests.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme:57]: My question remains though: does this change from `instrumented` set to `YES` to `NO` mean there will be no instrumented benchmark anymore?

Comment (maxep) [reply BenchmarkTests/BenchmarkTests.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme:57]: No, this is the env variable for running this app in Debug:
image

When set to `NO` (unselected), the `none` run will be selected. This is for development only, to simply select a run and a scenario, it is not applied to s8s build.

Comment (mariedm) [reply BenchmarkTests/BenchmarkTests.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme:57]: Got it, thanks!

**Pull request merged**